packages/bpk-component-calendar/src/date-utils.ts

```diff
@@ -97,7 +98,23 @@ function startOfWeek(
   return [date.getUTCFullYear(), date.getUTCMonth(), date.getUTCDate()];
 }
 
-function getCalendarMonthWeeks(date: Date, weekStartsOn: number) {
+function getCalendarNoCustomLabel(date: Date, weekStartsOn: number) {
+  return getCalendarMonthWeeks(date, weekStartsOn);
+}
+
+function getCalendar(
+  date: Date,
+  weekStartsOn: number,
+  formatDate: (d: Date) => Date | string,
+) {
+  return getCalendarMonthWeeks(date, weekStartsOn, formatDate);
+}
```

Comment:
Appreciate this could be cleaner but:
- I wanted to avoid separating the values into new state to avoid more refactoring and complexity to rationalise back together
- Mapping the values in after the initial generation is arguably slower and harder to read